cas-maestro.php

```diff
@@ -16,308 +17,356 @@
+        // add_action('retrieve_password', array(&$this, 'disable_function'));
+        // add_action('password_reset', array(&$this, 'disable_function'));
+        if ( $pagenow == 'profile.php' ) {
+          add_filter( 'show_password_fields', array( &$this, 'show_password_fields' ), 999 );
+        }
       } else {
-        $error = __("wpCAS is not configured. Please, login, go to the settings and configure with your credentials.", "CAS_Maestro");
-        //add_filter( 'login_head', array(&$this, 'display_login_notconfigured'));
+        $error = __( 'wpCAS is not configured. Please, login, go to the settings and configure with your credentials.', 'CAS_Maestro' );
+        // add_filter( 'login_head', array(&$this, 'display_login_notconfigured'));
       }
-
     }
-    add_action('wp_logout', array(&$this, 'process_logout'));
-
-    //Register the language initialization
-    add_action('init' ,array(&$this, 'lang_init'));
-    add_action('admin_init', array(&$this, 'add_meta_boxes'));
-    add_action('profile_update', array(&$this, 'onSaveProfile'),10,2);
-    add_action('admin_notices', array(&$this, 'notify_email_update'));
-
-    add_action('admin_menu', array( &$this,'register_menus'), 50);
-    add_action('admin_enqueue_scripts', array(&$this, 'register_javascript'));
-    //Filter to rewrite the login form action to bypass cas
-    if($this->bypass_cas) {
-      add_filter('site_url', array(&$this, 'bypass_cas_login_form'), 20, 3);
-      add_filter('authenticate', array(&$this, 'validate_noncas_login'), 30, 3);
+    add_action( 'wp_logout', array( &$this, 'process_logout' ) );
+
+    // Register the language initialization
+    add_action( 'init' ,array( &$this, 'lang_init' ) );
+    add_action( 'admin_init', array( &$this, 'add_meta_boxes' ) );
+    add_action( 'profile_update', array( &$this, 'on_save_profile' ),10,2 );
+    add_action( 'admin_notices', array( &$this, 'notify_email_update' ) );
+
+    add_action( 'admin_menu', array( &$this, 'register_menus' ), 50 );
+    add_action( 'admin_enqueue_scripts', array( &$this, 'register_javascript' ) );
+
+    // Filter to rewrite the login form action to bypass cas
+    if ( $this->bypass_cas ) {
+      $this->_write_log( __CLASS__ . '.' . __FUNCTION__ . ' (' . __LINE__ . ') : bypass_cas => ' . $this->bypass_cas );
+      add_filter( 'site_url', array( &$this, 'bypass_cas_login_form' ), 20, 3 );
+      add_filter( 'authenticate', array( &$this, 'validate_noncas_login' ), 30, 3 );
     }
   }
 
   /**
    * Initialize the language domain of the plugin
    */
   function lang_init() {
-     if (function_exists('load_plugin_textdomain')) {
-         load_plugin_textdomain( 'CAS_Maestro', false, dirname( plugin_basename( __FILE__ ) ) . '/lang/');
-     }
+    if ( function_exists( 'load_plugin_textdomain' ) ) {
+      load_plugin_textdomain( 'CAS_Maestro', false, dirname( plugin_basename( __FILE__ ) ) . '/lang/' );
+    }
   }
 
   /**
    * Generates a random string with a lenght
```

Comment:
the comment says "lenght" not "length", its not worth digging in just to fix it, but if you already are working on that file then its a quick fix. Chances are its spelled wrong in the original plugin.